xdsl/parser/affine_parser.py

```diff
@@ -121,6 +126,46 @@ def parse_expr() -> AffineExpr:
 
         return self.parse_comma_separated_list(self.Delimiter.PAREN, parse_expr)
 
+    # TODO: Extend to semi-affine maps
```

Comment:
issue?

Reply:
I'll check if it doesn't exist, it dates from the affine_map work

Reply:
And add the issue number on the TODO please ;)

Reply:
It exists; adding to the TODO
https://github.com/xdslproject/xdsl/issues/1087